lib/hoge.go

```diff
@@ -0,0 +1,5 @@
+package lib
+
+func GetStr() string {
```

Comment:
__[golint]__
* [golint] exported function GetStr should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->